app/extensions.js

```diff
@@ -400,6 +400,9 @@ module.exports.init = () => {
   }
 
   let loadExtension = (extensionId, extensionPath, manifest = {}, manifestLocation = 'unpacked') => {
+    if (extensionId === config.PDFJSExtensionId) {
+      manifestLocation = 'component'
```

Comment:
Note: 'component' type extensions are able to access cross-origin 'file:' URLs, but somehow they are not able to see 'file:' URLs in the chrome.webRequest listener.